content/en/docs/tutorials/pod-security/cluster-level-pss.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Applying Pod Security Standards at Cluster level
+content_type: tutorial
+weight: 10
+---
+
+{{% alert title="Disclaimer" %}}
+This tutorial applies only for new clusters.
+{{% /alert %}}
+
+# Pre-requisites
+
+Install the following on your workstation:
+
+- [KinD](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
```

Comment:
```suggestion
## {{% heading "objectives" %}}
```

Include a bulleted list of the objectives. What are the expected outcomes of this tutorial? 

For example: 

* Create a new cluster with Pod Security Standards applied
* Use `--dry-run` to experiment with cluster behavior when standards are applied
* Understand and use admission modes with Pod Security Standards